app/models/chef.rb

```diff
@@ -1,4 +1,6 @@
 class Chef < ApplicationRecord
    validates_presence_of :name
    has_many :dishes
+   has_many :dish_ingredients, through: :dishes
+   has_many :ingredients, through: :dish_ingredients
```

Comment:
I think you only included `has_many :dish_ingredients, through: :dishes` because you wanted to include this association: `has_many :ingredients, through: :dish_ingredients` to get to all the ingredients for a chef. You could have left off line 4 and just do something like this: `has_many :ingredients, through: :dishes`.

Dishes already have an association with dish ingredients so we don't have to create another association on the Chef model to get to the ingredients. This makes a lot more sense if you diagram it, but if it doesn't please reach out and I will explain it further.

I just want to mention that there's nothing wrong with having it, but if all we're trying to get to is the ingredients then we may not need it.